browser/src/control/Control.Notebookbar.js

```diff
@@ -332,12 +332,14 @@ L.Control.Notebookbar = L.Control.extend({
 		if (button.unoCommand && !isUnoCommand)
 			button.unoCommand = '.uno:' + button.unoCommand;
 
+		var imgUrl = button.imgurl.split('/')[1];
+
 		this.additionalShortcutButtons.push(
 			{
 				id: button.id,
 				type: 'toolitem',
 				text: button.label ? button.label : (button.hint ? _(button.hint) : ' '),
-				icon: button.imgurl,
+				icon: imgUrl,
```

Comment:
I think this is for custom icons which can come from outside COOL (integrator).
Then we need full URL.

In case of just standard COOL icons: we will use `command` to find icon -> so `icon:` can be empty.
If we want to use standard COOL icon without using `command`: client should provide good input in the PostMessage I think... 

Logic for that is here I think: https://github.com/CollaboraOnline/online/blob/master/browser/src/control/Control.JSDialogBuilder.js#L2287

Reply:
If button.imgurl is undefined it automatically uses the uno command's default icon. So no need to fix anything here.